docs/getting_started_guide/Installation_Options/Repository/Installing_SIMP_From_A_Repository.rst

```diff
@@ -12,8 +12,8 @@ your own internal VM stack, this is the method that you will want to use.
 
    This method does **not** modify your system's partitioning scheme or
    encryption scheme to meet any regulatory policies. If you want an example of
-   what that should look like either see the :ref:`simp-installation-guide` or
-   check out the `Kickstart`_ files in the `simp-core Git repository`_.
+   what that should look like see the :term:`Kickstart` files in the
+   `simp-core Git repository`_.
 
 
 Enable EPEL
```

Comment:
line 29:  'sudo yum install pygpgme yum-utils' needs '-y' at the end to match the presceding yum command